### ISSUE TITLE
default_role = 'guilabel' not functioning

### ISSUE THREAD

**Issue by daytonb** [sphinx-doc/sphinx#3960]
Subject: default_role = 'guilabel' not functioning

### Problem
- When I set `default_role = 'guilabel'` in conf.py the setting seems to have no effect. If I choose a different role just as `kbd` it works just fine.
- You can inspect the html output and see a `<span>` which should have the class `guilabel`, but it has no class.

#### Procedure to reproduce the problem
conf.py
```
default_role = 'guilabel'
```

index.rst
```
Click `OK` button.
```

Run `make clean && make html`.

#### Error logs / results
Produces no errors or warnings, just the following HTML.
```
<p>Click <span>OK</span> button.</p>
```

#### Expected results
```
<p>Click <span class="guilabel">OK</span> button.</p>
```

### Environment info
- OS:  Centos 7
- Python version: 2.7.5
- Sphinx version: 1.4.6

**Comment by daytonb** [comment]
Same issue with `menuselection` role.

**Comment by tk0miya** [comment]
Fixed at fa9ad85.
Thank you for reporting!
